README.mediawiki

```diff
@@ -937,7 +937,7 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Generic Signed Message Format
```

Comment:
You'll have to add `|- style="background-color: #ffcfcf"` two lines above this one

Reply:
Thanks, done